toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/GeoViewProxy.kt

```diff
@@ -17,10 +17,218 @@
+     * intersect this test region when rendered are returned, up to the [maximumResults] limit. A tolerance of
+     * 0 tests just the physical pixel at [screenCoordinate]. Tolerance values above 0 are in [Dp] and specify a
+     * circular region centered on [screenCoordinate], with radius equal to [tolerance]. The maximum allowed tolerance
+     * value is 100 [Dp], resulting in an identify circle of diameter 200 [Dp].
+     *
+     * The [returnPopupsOnly] parameter controls what properties are populated in the [IdentifyGraphicsOverlayResult]
+     * instances that are returned by the identify operation:
+     * * true: only [IdentifyGraphicsOverlayResult.popups] properties will be populated. Overlays
+     * without popups will be omitted.
+     * * false: [IdentifyGraphicsOverlayResult.graphics] properties will be populated. Overlays
+     * with popups will also return popups.
+     *
+     * @param screenCoordinate location on which to run identify in screen coordinates
+     * @param tolerance extent of the region used during the identify operation
+     * @param returnPopupsOnly whether the graphics property of the results are populated
+     * @param maximumResults maximum size of the result set of graphics to return. A null value indicates unlimited results
+     * @return A [Result] containing a [List] of [IdentifyGraphicsOverlayResult] containing one entry for each
+     * overlay in the view, or failure. Each entry holds a [GraphicsOverlay] and a [List] of [com.arcgismaps.mapview.Graphic]s
+     * @throws IllegalArgumentException if [maximumResults] is less than zero
+     * @since 200.4.0
+     */
+    public suspend fun identifyGraphicsOverlays(
+        screenCoordinate: ScreenCoordinate,
+        tolerance: Dp,
+        returnPopupsOnly: Boolean = false,
+        maximumResults: Int? = 1
+    ): Result<List<IdentifyGraphicsOverlayResult>> {
+        val maximumResultsSanitized = if (maximumResults == null || maximumResults < 0) {
+            -1
+        } else maximumResults
+        return geoView?.identifyGraphicsOverlays(
+            screenCoordinate,
+            tolerance.value.toDouble(),
+            returnPopupsOnly,
+            maximumResultsSanitized
+        ) ?: Result.failure(IllegalStateException(nullGeoViewErrorMessage))
+    }
+
+    /**
+     * Initiate an Identify operation on the specified [layer].
+     *
+     * The [tolerance] parameter determines the extent of the region used during the identify operation. Any GeoElements
+     * that intersect this test region when rendered are returned, up to the [maximumResults] limit. A tolerance of
+     * 0 tests just the physical pixel at [screenCoordinate]. Tolerance values above 0 are in [Dp] and specify a circular
+     * region centered on [screenCoordinate], with radius equal to [tolerance]. The maximum allowed tolerance value is 100 [Dp],
```

Comment:
Linking to [Dp] once in the doc is probably enough:
```suggestion
     * region centered on [screenCoordinate], with radius equal to [tolerance]. The maximum allowed tolerance value is 100 Dp,
```

Would need to be updated in other places as well.